File: joueur/Inventaire.java
package joueur;

import java.util.HashMap;
import java.util.LinkedHashMap;

public class Inventaire {
    
    private static HashMap<Integer, Object> hInventaire = new LinkedHashMap<>();

    public static void addObjet(Object obj)
    {
        hInventaire.put(hInventaire.size() + 1, obj);
    }



    public static HashMap<Integer, Object> gethInventaire() {
        return hInventaire;
    }

    public static Object getObjet(int i)
    {
        return hInventaire.get(i);
    }
}

File: Cs/joueur/Inventaire.cs
using System.Collections.Generic;

namespace joueur
{
    public class Inventaire
    {
        private static Dictionary<int, object> inventaire = new Dictionary<int, object>();

        public static void AddObjet(object obj)
        {
            int nextKey = inventaire.Count + 1;
            inventaire[nextKey] = obj;
        }

        public static Dictionary<int, object> GetInventaire()
        {
            return inventaire;
        }

        public static object GetObjet(int i)
        {
            inventaire.TryGetValue(i, out var obj);
            return obj;
        }
    }
}
